### DISCUSSION TITLE
What is the meaning of "NOT_SET" when starting a server?

### DISCUSSION THREAD

**Discussion by Amenel** [GameServerManagers/LinuxGSM#4274] [Q&A] [answered]
Hello Community, I salute you! I have been running the Left 4 Dead community servers for several years and for a lower number of years, there's something that has been bugging me bu I never took the time to investigate beyond what I did when it started bugging me. Here's the thing: when I start or restart a Left 4 Dead server with LGSM, the executable (which has been renamed to `l4dserver01`) outputs this:

```
[l4d@l4dcommunity:~]$ ./l4dserver01 restart
[  OK  ] Stopping l4dserver01: Graceful: CTRL+c: 1: OK
[  OK  ] Starting l4dserver01: NOT SET
```

I am wondering what "NOT SET" refers to in this specific line. I suppose that there is some optional configuration value that could be set but it hasn't been set (by me) and therefore, LGSM, not knowing the contents of the value, defaults to "NOT SET".
If that's the case, I would like to know which configuration value or script variable that is. That's why I'm coming to you for some knowledge or help: maybe that someone knows where I could set that value.
Is it the server name? Because when I type `l4dserver01 dt`, I get some text that is not "NOT SET", which leaves me confused.

Note that I have been running two L4D servers off the same VPS for all those 5+ years and each server has its own executable and its own configuration. To put it simply, the two servers never collided once I got them running.

If my supposition is wrong, I'd like to be corrected :)
Thanks.

**Comment by Pshock13** [comment, marked as answer]
I believe what you are experiencing is that of which is mention in the issue here: https://github.com/GameServerManagers/LinuxGSM/issues/4222

I would urge you to update LGSM as this issue has been resolved.

**Comment by Amenel** [comment]
Issue #4222 does match my question and I gather from its discussion thread that "NOT SET" is therefore for the server name. And this does answer my question. Thanks.

My LGSM installation is on auto-update via a daily cronjob and it's running the latest version, i.e. v23.4.0, yet it reports "NOT SET".

Using `&#160;` instead of spaces in `servername` or `gdname` or `hostname` did not help either.
